scripts/cli/start_cli_testnet.sh

```diff
@@ -14,15 +14,15 @@ SCRIPT_PATH="$(dirname $0)"
 RUN_PARAMS="--host ac.testnet.libra.org --port 8000 -s $SCRIPT_PATH/consensus_peers.config.toml"
 
 case $1 in
-    -h | --help)
-        print_help;exit 0;;
-    -r | --release)
-        echo "Building and running client in release mode."
-        cargo run -p client --release -- $RUN_PARAMS
-        ;;
-    '')
-        echo "Building and running client in debug mode."
-        cargo run -p client -- $RUN_PARAMS
-        ;;
-    *) echo "Invalid option"; print_help; exit 0;
+	-h | --help)
+		print_help;exit 0;;
+	-r | --release)
+		echo "Building and running client in release mode."
+		cargo run -p client --release -- $RUN_PARAMS $FLAGS
+		;;
+	'')
+		echo "Building and running client in debug mode."
+		cargo run -p client -- $RUN_PARAMS $FLAGS
+		;;
+	*) echo "Invalid option"; print_help; exit 0;
 esac
```

Comment:
Ok so here's what we can do. Have a first pass which does setting of flags and consuming some of the args, then we can pass in "$@", so something sort of like this:

```
while
do
	case "$1" in
	-h | --help)
		print_help;exit 0;;
	-r | --release)
		RELEASE=1
		;;
	--)
		shift
		break
		;;
	*) echo "Invalid option"; print_help; exit 0;;
	esac
	shift
done


if $RELEASE; then

        echo "Building and running client in release mode."
        cargo run -p client --release -- $RUN_PARAMS "$@"
else
        echo "Building and running client in debug mode."
        cargo run -p client -- $RUN_PARAMS "$@"
        ;;
fi
```

Reply:
Then you can just pass all the args after `--` to the cli directly

Reply:
:+1: a fine suggestion.  Made some tweaks, PTAL.